ejcIMC.py

```diff
@@ -25,6 +25,24 @@ def imcCalculate(statureValue, weigthValue):
     IMC = (weigthValue/(statureValue**2))
     return IMC
 
+def yourImc(imc):
+    if imc <= 16:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez severa')
+    elif imc <= 16.99:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez moderada')
+    elif imc <= 18.49:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez aceptable')
+    elif imc <= 24.99:
```

Comment:
24.99 : is a magic number.... try to do this variable declaration begining the code.... In variables section
and explain what do u refer to that.

In the future code maybe need to change that values.... it is better to change them at the beginning of the document than to be forced to read the whole project